vtex-apps/order-placed
# Fix/add free tag price for gifts

#### What is the purpose of this pull request?


This PR replaces the price on the order list with a label `FREE` when the product is a gift. It relies on the `isGift`value from the `OrderItem` element received in the `Product` component. 

#### What problem is this solving?

This change was requested by Whirlpool to the project team in WE.

#### How should this be manually tested?

[This link](https://fila--itwhirlpoolqa.myvtex.com/checkout/orderPlaced/?og=1107482689939) takes you (if logged as superAdmin) to an order that contains a gift. Instead of the price item, you should see __FREE__.

#### Screenshots or example usage

<img width="1432" alt="Screen Shot 2021-03-30 at 3 38 25 PM" src="https://user-images.githubusercontent.com/38737958/113001123-cad12180-9170-11eb-9d5e-adb7ee49ca2a.png">


#### Types of changes
